tests/MongoDB.Driver.Tests/Specifications/crud/CrudTestRunner.cs

```diff
@@ -136,12 +146,27 @@ private void DropCollection(string databaseName, string collectionName)
 
         private void ExecuteOperation(IMongoClient client, IMongoDatabase database, IMongoCollection<BsonDocument> collection, BsonDocument operation, BsonDocument outcome, bool async)
         {
+            JsonDrivenHelper.EnsureAllFieldsAreValid(operation, "name", "object", "collectionOptions", "arguments", "error", "result");
+
+            if (operation.TryGetValue("object", out var @object))
+            {
+                if (@object.AsString == "database")
+                {
+                    collection = null;
+                }
+            }
+
             var name = (string)operation["name"];
             var test = CrudOperationTestFactory.CreateTest(name);
             bool isErrorExpected = operation.GetValue("error", false).AsBoolean;
 
             var arguments = (BsonDocument)operation.GetValue("arguments", new BsonDocument());
             test.SkipIfNotSupported(arguments);
+            if (operation.TryGetValue("result", out var result))
+            {
+                outcome = outcome ?? new BsonDocument();
+                outcome["result"] = result;
```

Comment:
This is a hack.

These JSON tests are also in a state of flux. The new db-aggregate.json file uses a newer format with an explicit "result" field. The other JSON test files have the "result" embedded in the "outcome".

The quick and dirty fix is to simply plant the "result" where the code already expects it.

All our JSON test runners need to be refactored some day.

My goal for today is to make the smallest change possible that allows running the new db-aggregate.json tests.